api/src/plugins/jwt.ts

```diff
@@ -26,11 +44,12 @@ const jwt: FastifyPluginCallback = async (fastify, opts, done) => {
   fastify.decorate("auth", async (request: Request, reply: Reply) => {
     try {
       const { token } = request.cookies;
```

Comment:
It's undefined as you pass it in headers as auth token, the token isn't in the cookies anymore. It seems to be in `request.headers.authorization`